wsclient/src/main/java/org/hpccsystems/ws/client/utils/Connection.java

```diff
@@ -486,12 +486,27 @@ private void setProtocol(String protocol_)
      * @param port_
      *            the new port
      */
-    private void setPort(String port_)
+    public void setPort(String port_)
```

Comment:
@jpmcmu I'm making this change so we can copy the wseclwatch connection, and simply edit the port to the wsecl port.
I'm not sure if we'd be better off creating a brand new connection based off of the eclwatch connection

@asselitx ignore this change